glue/core/visual.py

```diff
@@ -25,20 +25,22 @@ class VisualAttributes(HasCallbackProperties):
 
     """
 
-    def __init__(self, parent=None, washout=False, color=None, alpha=None):
+    def __init__(self, parent=None, washout=False, color=None, alpha=None, preferred_cmap=None):
 
         super(VisualAttributes, self).__init__()
 
         # We have to set the defaults here, otherwise the settings are fixed
         # once the class is defined.
         color = color or settings.DATA_COLOR
         alpha = alpha or settings.DATA_ALPHA
+        preferred_cmap = preferred_cmap
```

Comment:
Is this line needed?

Reply:
Not really. Was thinking about doing something extra about it which turned out not necessary after all. But forgot to remove it. Thanks for pointing this out!